types/node/globals.d.ts

```diff
@@ -85,13 +85,6 @@ interface Console {
     warn(message?: any, ...optionalParams: any[]): void;
 
     // --- Inspector mode only ---
-    /**
-     * This method does not display anything unless used in the inspector.
-     *  The console.markTimeline() method is the deprecated form of console.timeStamp().
-     *
-     * @deprecated Use console.timeStamp() instead.
-     */
-    markTimeline(label?: string): void;
```

Comment:
I'm not sure about `markTimeline`, `timeline` and `timelineEnd`. Doc says there are only available if inspector is enabled and I don't see them in repl. doc tells the same for `profile` and `profileEnd` but they are present.

Reply:
They don't exist in either case, see https://github.com/nodejs/node/pull/30346